webpack-watcher.js

```diff
@@ -38,10 +39,46 @@ module.exports = (options) => {
       },
       whenDone: () => new Promise((resolve, reject) => {
          watchers.add({resolve, reject});
-         child.send({event: 'isRunning'})
+         if (child.connected) {
+            child.send({event: 'isRunning'})
+         } else {
+            notifyWatchers("Webpack child process crashed, check the log");
```

Comment:
Could we use `options.logPath` to point people directly at the log and _not_ suggest checking the log if `options.logPath` is falsey?

Reply:
Fixed.